framework/src/main/java/fi/otavanopisto/pyramus/koski/model/lukio/LukionOppiaineenArviointi.java

```diff
@@ -9,6 +9,9 @@
 
 public class LukionOppiaineenArviointi {
 
+  public LukionOppiaineenArviointi() {
```

Comment:
This one doesn't need JsonDeserialize(using = JsonDeserializer.None.class)?

Reply:
Nope, it's only for subclasses where parent class has a custom deserializer.